### PR TITLE
[Orc8r] Add changes to pass restricted plmn IDs and service area maps to mconfig

### LINKED ISSUE #4841: Orc8r API support

### PR DESCRIPTION
**Pull request by karthiksubraveti**
Signed-off-by: Karthik Subraveti <ksubraveti@fb.com>

## Summary
This change copies over the configs to mconfigs. 

## Test Plan
Added tests which verify that the mconfigs contain restricted plmns and service area maps.
I was also able to manually verify that mconfig comes down to the gateway with restricted_plmn information

## Additional Information

- [ ] This change is backwards-breaking

### CODE REVIEW

**Review by ulaskozat — APPROVED** [review]
LGTM

File: lte/cloud/go/services/lte/servicers/builder_servicer_test.go [hunk 1]
```diff
@@ -185,6 +185,31 @@ func TestBuilder_Build(t *testing.T) {
...
+	actual, err = build(&nw, &graph, "gw1")
+	assert.NoError(t, err)
+	assert.Equal(t, expected, actual)
+
+	//verify service area map
+	setEpcNetworkServiceAreaMap(&nw, map[string]lte_models.TacList{"001": []lte_models.Tac{111, 112}})
+	mmeVals.ServiceAreaMaps = map[string]*lte_mconfig.MME_TacList{
+		"001": {
+			Tac: []uint32{111, 112},
+		},
```
Comment (ulaskozat) [inline lte/cloud/go/services/lte/servicers/builder_servicer_test.go]: as a nit, maybe adding multiple entries in the test case would be better.

Comment (mpgermano) [reply lte/cloud/go/services/lte/servicers/builder_servicer_test.go]: +1 for this

**Review by mpgermano — APPROVED** [review]

File: lte/cloud/go/services/lte/servicers/builder_servicer_test.go (hunk 1 above)
Comment (karthiksubraveti) [reply lte/cloud/go/services/lte/servicers/builder_servicer_test.go]: Done. Thanks for the review.

**Pull request merged**